app/src/crypto.c

```diff
@@ -224,15 +229,15 @@ zxerr_t crypto_sign_sr25519(uint8_t *signature, uint16_t signatureMaxlen,
         CATCH_ALL
         {
             CLOSE_TRY;
-            return zxerr_unknown;
+            err = zxerr_unknown;
```

Comment:
similar to previous comment

Reply:
https://github.com/Zondax/ledger-edgeware/blob/b1a1285f78b5ebe1b81a4471b6424a916f09f750/app/src/crypto.c#L153-L157 
Should we only set the error and *signatureLen = 0 here too @ftheirs ?

Reply:
ohh yeap! Good catch 😄